virt/install/uninstalling-virt.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-virt"]
+include::_attributes/common-attributes.adoc[]
+= Uninstalling {VirtProductName}
+:context: uninstalling-virt
+
+toc::[]
+
+You can uninstall {VirtProductName} by using the {product-title} xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console] or the command line interface (xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[CLI]).
```

Comment:
Maybe this is a virt convention, but in uninstall modules, I tend to link the _you can uninstall by using the web console or CLI_ to the uninstall sections. Feel free to ignore.